src/modules/dashboard/components/ExpenditurePage/Recipient/LockedRecipient.css

```diff
@@ -0,0 +1,150 @@
+.settingsRow {
+  composes: flexContainerRow flexAlignCenter flexJustifyBetween from '~styles/layout.css';
+  min-height: 41px;
+}
+
+.settingsRow button[aria-expanded="true"] {
+  padding: 2px 0 2px 6px;
+  border-radius: 4px;
+  background-color: rgb(229, 230, 247);
```

Comment:
Can we please replace all instances of this color with this variable `--grey-blue-1`